public/components/custom_panels/redux/panel_slice.ts

```diff
@@ -212,40 +217,68 @@ const deleteLegacyPanels = (customPanelIdList: string[]) => {
 };
 
 export const deletePanels = (panelsToDelete: CustomPanelType[]) => async (dispatch, getState) => {
-  const ids = panelsToDelete.map((p) => p.id);
-  await Promise.all([deleteLegacyPanels(ids), deletePanelSO(ids)]);
+  const toastMessage = `Observability Dashboard${
```

Comment:
I'm finding myself re-reading this block every time to see what it does... and that it does it right. *sigh*

Can we wrap this in a function `pluralize()` or something?

Maybe even simpler... just say "Observability Dashboard(s)".   No one truly cares... unless UX objects on some principle reason.